its-incident-app/micro/auth-core/Makefile

```
@@ -0,0 +1,57 @@
+build:
+	@echo "Building $(SERVICE)..."
+	docker build --platform=linux/amd64 \
+		-t $(IMAGE_PREFIX)/$(SERVICE):latest .
+
+# Dockerイメージをプッシュ
+push:
+	@echo "Pushing $(SERVICE)..."
+	docker push $(IMAGE_PREFIX)/$(SERVICE):latest
+
+# サービスをデプロイ
+deploy:
```

checkmake (0.2.2)

[warning] Target body for "deploy" exceeds allowed length of 5 (7).

(maxbodylength)